docs/Параметризация.md

```
@@ -4,44 +4,173 @@
+**Параметры для поля класса, полученные из функции**
+
+Используйте аннотацию `&ИсточникПараметров` для указания функции, возвращающей коллекцию значений для поля. Функция должна быть определена в классе бенчмарка.
+
+```bsl
+&ИсточникПараметров("ПолучитьАлгоритмыХеширования")
+Перем АлгоритмХеширования Экспорт;
+```
+
+```bsl
+Функция ПолучитьАлгоритмыХеширования() Экспорт
+	Значения = Новый Массив();
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)